mesonbuild/compilers/fortran.py

```diff
@@ -273,6 +271,14 @@ def _get_trials_from_pattern(cls, pattern, directory, libname):
     def _get_file_from_list(files) -> List[str]:
         return CCompiler._get_file_from_list(files)
 
+    @classmethod
+    def _get_trials_from_pattern(cls, pattern, directory, libname):
+        return CCompiler._get_trials_from_pattern(pattern, directory, libname)
+
+    @staticmethod
```

Comment:
__[Flake8]__ [F811] redefinition of unused '_get_file_from_list' from line 270

<sub>posted by Sider</sub>